src/packages/searchbar/searchbar.taro.tsx

```
@@ -55,137 +64,241 @@ export const SearchBar: FunctionComponent<
+  const handleFocus = useCallback(
+    (event: BaseEventOrig<InputProps.inputForceEventDetail>) => {
+      onFocus && onFocus(event.detail?.value, event)
+      setInnerTag(false)
+    },
+    [onFocus]
+  )
+
+  const handleBlur = useCallback(
+    (event: BaseEventOrig<InputProps.inputValueEventDetail> | any) => {
+      searchInputRef.current?.blur()
+      onBlur && onBlur(event.detail?.value, event)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)